integrations/tensorflow/build_tools/bazel/iree-tf.bazelrc

```diff
@@ -11,6 +11,10 @@ try-import %workspace%/../../build_tools/bazel/iree.bazelrc
 # to the OSS codebase.
 build --nocheck_visibility
 
+// TF uses C++ 17
```

Comment:
Can you call this out in the description? That's a somewhat major change

Reply:
well this didn't solve the problem so it might not even be needed. Still trying to figure out why TF build fail. Any suggestion is appreciated

Reply:
Wrong comment syntax

```suggestion
# TF uses C++ 17
```